docs/.vuepress/enhanceApp.js

```diff
@@ -33,9 +33,31 @@ const buildActiveHeaderLinkHandler = () => {
   };
 };
 
+const themeLoader = () => {
```

Comment:
I would like to keep the `enchenceApp.js` small as possible. Could you extract it?

Reply:
Done in https://github.com/handsontable/handsontable/pull/8339/commits/9e120933fd6219df1b2d6dfccabf6184d326ee6a